parsing/src/processor.ts

```diff
@@ -180,6 +199,8 @@ function getProducingBuildings(data: any[]): string[] {
                 if (producedInBuildings) {
                     producedInBuildings.forEach((buildingName: string) => producingBuildingsSet.add(buildingName));
                 }
+            } else if (entry.ClassName === "Desc_NuclearWaste_C") { 
```

Comment:
Any reason why we can't just add this in at the end of the processing rather than running the conditional against every entry?

Reply:
Definitely focused on a 'build, then optimize' methodology. The code runs in less than a second. Now that it works, I want to focus on tests and optimizing. In Particular, I think some of these if's can be removed with some tests and optimization.

Reply:
Aight I'll let this one go for now then, you're right premature optimisation is the root of all evil as they say